client/src/components/productDetail/ProductRecommend/index.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+
+import React, { FC } from 'react';
+import { useHistory } from '~/core/Router';
+import { ImageWrapper, ProductRecommendContainerWrapper } from './index.style';
+
+interface Props {
+  products: {
+    idx: number;
+    thumbnail: string;
+  }[];
+}
+
+const ProductRecommendContainer: FC<Props> = ({ products }) => {
+  const { push } = useHistory();
+
+  const handleImgClick = (idx: number) => {
+    push(`/products/${idx}`);
+  };
```

Comment:
useCallback으로 묶어볼까요...!?
```suggestion
  const handleImgClick = useCallback((idx:number) => { 
    push(`/products/${idx}`);
  , []};
```